### ISSUE TITLE
Problems installing clean from R 4.3.2

### ISSUE THREAD

**Issue by grwhumphries** [dmpstats/sCRM#5]
Looks like there's some problems with installing clean from R 4.3.2. 

First is that mran isn't being recognized for some of the packages installs, getting a "URL error" message. (includes the promises package).  

covrpage is also not installing correctly

Finally, getting an error with the path names:   "Files in 'inst' are installed at top-level;  Error in system.file, paths can't start with 'inst'"

**Comment by bcaneco** [comment]
Yes, that's not surprising given the app-package was developed under R 4.2.2 and it hasn't been updated to the latest 4.3.x. 

If you're trying to have a quick peak inside, maybe `renv::restore()` is your friend?

**Comment by grwhumphries** [comment]
Yeah - there was a big leap between 4.2 and 4.3 with a number of packages.  renv::restore() isn't doing it's job either as it's looking for mran links during the install which don't seem to be working.

**Comment by bcaneco** [comment]
Almost exactly a year later - but there's a good chance this issue has been resolved in the latest sCRM release: [v0.2.1](https://github.com/dmpstats/sCRM/releases/tag/v0.2.1).

In-house testing shows the app runs locally as expected from a fresh install on R 4.4.3.

Just a heads-up: make sure to run `library(sCRM)` before calling `run_app()`, so the default package data is properly loaded into your environment.

I'll close this issue for the moment, but feel free to re-opening it if the problem is still persistent.
